.appveyor.yml

```diff
@@ -16,6 +16,7 @@ install:
   - npm ci
   - npm install --global codecov
   - npm install eslint-plugin-node@latest --save-dev
+  - npm install eslint-plugin-prettier@latest --save-dev
```

Comment:
This didn't work. Is the travis one the same?

Reply:
Travis did not complain about this

Reply:
windows specific issue?